runner.py

```diff
@@ -0,0 +1,149 @@
+import argparse
```

Comment:
Same as for `make_datasets.py`, `runner.py` should have a copyright header at the top, but with only 2020.

Reply:
Header was added